client/lib/main.js

```diff
@@ -56,7 +56,35 @@ function main(options, callbacks) {
     // TODO: Get URI from mappings.
     uri = "http://localhost:8080/bundles/main.js";
   }
-  SANDBOX(uri, function(sandbox) {
+  SANDBOX(uri, {
+    global: {
+      // Not available in this context.
+      Components: undefined
+    },
```

Comment:
The firebug sdk uses a global that we need to inject into the sandbox: https://github.com/firebug/firebug.sdk/blob/3f012346e617033fef1381cac0f1f4174a906beb/lib/core/actor.js#L18